Test/FloatTorchTensorUnitTest.cs

```diff
@@ -0,0 +1,99 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using TorchSharp;
+
+namespace Test
```

Comment:
Maybe use `TorchSharp.Tests` or such as the namespace and project name?

Reply:
Fixed